### PR TITLE
Disable database cleanup by default

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [ ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

**Review by ivanfetch-wt — APPROVED** [review]
As long as our Helm values for prod are updated *first* to include `databaseCleanupCronjob.enabled=true`, this PR looks good.

I tested to make sure the template renders and uses the new value.

File: stable/fairwinds-insights/values.yaml [hunk 1]
```diff
@@ -284,6 +284,8 @@ emailCronjob:
     runAsUser: 10324
 
 databaseCleanupCronjob:
+  # -- Enable database cleanup
+  enabled: false
```
Comment (makoscafee) [inline stable/fairwinds-insights/values.yaml:288]: BTW, do we need to enable this in `ci/test.yaml`?

**Review by makoscafee — APPROVED** [review]

File: stable/fairwinds-insights/values.yaml (hunk 1 above)
Comment (jdesouza) [reply stable/fairwinds-insights/values.yaml:288]: that's a good idea

File: stable/insights-agent/ci/test-values.yaml [hunk 2]
```diff
@@ -59,3 +59,6 @@ awscosts:
   tagkey: kubernetes_cluster
   tagvalue: staging.internal.reactiveops.com
   workgroup: cur_athena_workgroup
+
```
Comment (jdesouza) [inline stable/insights-agent/ci/test-values.yaml]: @makoscafee right?

Comment (makoscafee) [reply stable/insights-agent/ci/test-values.yaml]: yes that looks about right

File: stable/fairwinds-insights/ci/test-values.yaml [hunk 3]
```diff
@@ -48,6 +48,7 @@ emailCronjob:
   schedules: []
 
 databaseCleanupCronjob:
+  enabled: true
```
Comment (makoscafee) [inline stable/fairwinds-insights/ci/test-values.yaml:51]: actually just to think of it ... you do not need this. none of the cronjob are enabled here

**Comment by rbren** [comment]
@jdesouza thanks for the fix! However, I think we should revert this PR, and make the change in the insights-release repo - we want to make sure our self-hosted users have this job on by default.

**Pull request merged**